application/config/development.py

```diff
@@ -2,8 +2,8 @@
 
 ADMIN_HOST = 'https://admin-beta.development.performance.service.gov.uk'
 
-BACKDROP_HOST = 'http://www.development.performance.service.gov.uk'
-STAGECRAFT_HOST = 'http://stagecraft.development.performance.service.gov.uk'
+BACKDROP_HOST = 'http://localhost:3039'
+STAGECRAFT_HOST = 'http://localhost:3204'
```

Comment:
Port numbers are different, as spoken about we should just abandon trying to support the deprecated pp-puppet VM and just start using the GOV.UK dev vm